src/backends/policy_engine/souffle/check_policy_compliance_test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
```

Comment:
Should we have a copyright header here? Also maybe a quick comment describing what this file does?

Reply:
done